### PR TITLE
Handle the two different orgs now

### PR DESCRIPTION
**Pull request by bloodearnest**
- enable automatic branch deletion
- add editors team for non-study repos in the opensafely org

### CODE REVIEW

File: manage-github.py [hunk 1]
```diff
@@ -8,6 +9,29 @@
 
 import client
 
+
+# These objects define the policies for ou
```
Comment (ghickman) [inline manage-github.py:13]: `ou`?

**Review by ghickman — APPROVED** [review]

**Review by evansd — APPROVED** [review]
Declarative config ftw!

File: manage-github.py (hunk 1 above)
Comment (bloodearnest) [reply manage-github.py:13]: `ou` is french for typo

Comment (madwort) [reply manage-github.py]: TIL

**Pull request merged**